Sources/Html/XmlRender.swift

```diff
@@ -0,0 +1,15 @@
+/// Renders an array of nodes to an XML string.
+///
+/// - Parameter nodes: An array of nodes.
+public func xmlRender(_ nodes: [Node]) -> String {
```

Comment:
What do you think of keeping this "private" for now while we consider some alternatives?
```suggestion
public func _xmlRender(_ nodes: [Node]) -> String {
```
I was thinking we could introduce some kind of helper strategy functions that handle this stuff...
``` swift
render(nodes, as: .html5)
// ...or...
render(nodes, as: .xml)
```

Reply:
ah yeah I'm down to do that so that we can explore more API's of this soon. will do!

Reply:
ah yeah I'm down to do that so that we can explore more API's of this soon. will do!